### ISSUE TITLE
Only keep mitigated items from scan reports, and don't reactivate them

### ISSUE THREAD

**Issue by Gby56** [DefectDojo/django-DefectDojo#6892]
**Is your feature request related to a problem? Please describe**
This problem comes with specific scanners. Mostly, scanners that do no have mitigation timestamps, or a triage system.
They will always return all findings, and the only way to remove them is to add comments in your source etc...

The problem with that is that defectdojo's reimport API will keep reactivating findings, even if you triaged things in Defectdojo.
If the scan has "mitigated" timestamps it could be easy to understand where the finding has been mitigated first: defectdojo or your scanner. But this isn't the case and that's why the idea is the following:

**Describe the solution you'd like**
Add a `Do not reactivate` check box in the reimport form, with a subtext like `"Select if the import should ignore active findings from the report, and keep existing findings closed, without reactivating them."`

When checked, the parsing will ignore incoming active findings that match a closed finding in Defectdojo. This prevents reactivating them for no reason, and you can set that option for the "dumb" scanners that give reports without enough information.

Basically, it will enable users to accumulate triage information without reopening findings. Even for "smarter" scanners that including triage information, it can be useful to basically let Defectdojo override any opened issue coming from a scanner, if the triage was done once, it will stay like that. (Can be dangerous if your scanner will reopened issues legitimately...)

**Describe alternatives you've considered**
There is no alternative, because semgrep, tfsec and other scanners don't have that information in their report. You need to persist the triage information otherwise it can reopen the findings for no reason.

**Comment by manuel-sommer** [comment]
The PR was merged. Can we close this @mtesauro?
